backend/src/database/migrations/20200622182413-create-usuario.js

```diff
@@ -0,0 +1,21 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Usuarios', {
+    id: {
+      allowNull: false,
+      autoIncrement: false,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    usuario: {
+      type: Sequelize.STRING,
```

Comment:
Querríamos que los campos de usuario, nombre y email sean únicos y creo que también querríamos que no sean nulos, podríamos agregar los constraints.

```suggestion
      type: Sequelize.STRING,
      allowNull: false,
    unique: true,
```